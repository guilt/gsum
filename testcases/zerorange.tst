Name: zerorange
Description: Hashing the first byte of a zero-filled file
File: zerorange.bin 16B zero
AlgoMatrix: default
Range: 0-1
