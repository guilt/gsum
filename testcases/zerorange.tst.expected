Hash-default-0-1: 5df6e0e2761359d30a8275058e299fcc0381534545f55cf43e41983f5d4c9456
